docs/source/losses.rst

```diff
@@ -10,3 +10,11 @@ This package lists common losses across research domains
 Your Loss
 ---------
 We're cleaning up many of our losses, but in the meantime, submit a PR to add your loss here!
+
+----------------------------------------
```

Comment:
also, this line shall have a fixed length, pls check either HR in docs

Reply:
I am not sure exactly what you mean on this one. Is this not supposed to be used or is there a specific length for seperators? I have made an update, let me know if its correct